package.json

```diff
@@ -41,7 +42,8 @@
     "core-js": "^2.4.1",
     "es6-promise": "^4.0.3",
     "kinvey-node-sdk": "3.2.2",
-    "lodash": "^4.16.2"
+    "lodash": "^4.16.2",
+    "wdio-spec-reporter": "0.0.3"
```

Comment:
@thediycoder should we rather move this to `devDependencies`? We've had problems in the past with dependency managers (nexus).